JapaneseFilter/sections/specific.txt

```diff
@@ -13,7 +13,7 @@
 ! ##.pr
 ceron.jp,kids.gakken.co.jp,5chan.jp,toyokeizai.net,twpf.jp,chunichi.co.jp,tokyo-np.co.jp,db.msin.jp,ladiesfashion.jp,minpo.jp##.pr
 ! ##.bnr
-npn.co.jp,afpbb.com,2ch.pet,hamakei.com,keizai.biz,minkei.net,shibukei.com##.bnr
+baby-calendar.jp,npn.co.jp,afpbb.com,2ch.pet,hamakei.com,keizai.biz,minkei.net,shibukei.com##.bnr
```

Comment:
Hmm, blocks app banner on `https://baby-calendar.jp/recipe/5739` too on mobile. @zloyden What do you think? I guess it's acceptable.

Reply:
Can you show a screenshot? I don't see it on my end.